src/test/suite/editors/playgroundController.test.ts

```diff
@@ -79,49 +86,70 @@ suite('Playground Controller Test Suite', () => {
     });
```

Comment:
Small thing but I think for this test to be a bit more catch all we should test that the error message is what we expect it to be.